0_download_data/correct_images.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Correct Second Plate NF1 Images
+
+# In[1]:
+
+
+import pathlib
+import importlib
+correct = importlib.import_module("correctionutils")
```

Comment:
Consider using an `import` over `import_module` if it makes sense to do so here (and also removing the line importing importlib). For example:
```suggestion
import correctionutils as correct
```

Reply:
Woaaah, I never knew this existed. That is so much neater! Changed!